docs/en/observability/synthetics-get-started-project.asciidoc

```diff
@@ -47,6 +47,17 @@ You can run {project-monitors} on one or both of the following:
   To use {private-location}s you must create a {private-location} before continuing.
   For step-by-step instructions, refer to <<synthetics-private-location>>.
 
+// Should we mention the possibility of not being able to run monitors on public locations
+// here or is disabling public locations rare enough that it's not worth adding more complexity
+// to the getting started guide? If they see an error or warning, they should be able to
+// diagnose the issue by viewing the troubleshooting doc.
+
+[NOTE]
+====
+Administrators can restrict access to Elastic's global managed testing infrastructure.
+Read more in <<disable-managed-locations,writer role documentation>>.
+====
```

Comment:
Should we mention the possibility of not being able to run monitors on public locations here or is disabling public locations rare enough that it's not worth adding more complexity to the getting started guide? If they see an error or warning, they should be able to diagnose the issue by viewing the [troubleshooting doc](https://observability-docs_3305.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-troubleshooting.html#synthetics-troubleshooting-no-locations).

Reply:
Good question.  Maybe it's just worth adding a link "if you can't see any locations here, see the troubleshooting guide" to help it not being a dead end, relying on the user to see there are troubleshooting docs